src/Client/GameClient.cpp

```diff
@@ -299,3 +320,23 @@ void GameClient::changeLevel(GameMsgLevelChange nextLevelConfig) {
 void GameClient::setServerDown() {
     serverIsDown = true;
 }
+
+void GameClient::pauseSoundEffects(int music, int sounds) {
+    if (music && !musicManager->isMusicPaused()){
```

Comment:
I don't really like the double check, maybe it would be more performant (or nice) doing smt like this: 
```suggestion
    if (music) {
        if (musicManager->isMusicPaused()) {...}
        else {...}
```

Reply:
Te lo cambio, pero no digas performante porque es UN IF hdp, es O(1) de aca a la china en cualquier caso